ckanext/gla/public/gla.css

```diff
@@ -1198,9 +1212,9 @@ a.tag:hover {
 
 .filters {
     padding-top: 20px;
-    position: relative!important;
-    background: transparent!important;
-    display: block!important;
+    position: relative !important;
```

Comment:
Don't think we need all the filters stuff since we removed the filter panel, but I see no reason not to fix bad css